apple-nav/src/components/SubNav.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { keyframes } from 'styled-components';
+
+const fadeIn = keyframes`
+from {
+	opacity: 0;
+}
+
+to {
+	opacity: 1;
+}
+`;
+
+const Section = styled.section`
+	background: grey;
+	opacity: 1;
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-around;
+	align-items: center;
+	padding: 2.5% 5%;
+	animation: ${fadeIn} 2s;
+`;
+
+const P = styled.p`
+	text-align: center;
+	margin: 20%;
+	font-size: 4rem;
+	font-weight: bold;
+	font-family: cursive;
+	color: black;
+`;
+const SubNav = (props) => {
+	const category = props.match.params.category;
+
+	return (
+		<React.Fragment>
+			{props.list.slice(1, 7).map((item, index) =>
+				item.name === category ? (
+					<React.Fragment>
```

Comment:
The key always needs to be in the parent of whatever your looping or mapping over. So if you moved the `key={index` that you did on line 43, inside this fragment like this `<React.Fragment key={index}>`, you would get rid of that error message in the console.